ldk/javascript/examples/self-test-loop/src/tests/index.ts

```diff
@@ -251,14 +251,14 @@ export const testMarkdownWhisper = (): Promise<boolean> =>
             type: whisper.WhisperComponentType.RadioGroup,
           },
           {
-            alignment: whisper.Alignment.SpaceEvenly,
+            justifyContent: whisper.JustifyContent.SpaceEvenly,
             direction: whisper.Direction.Horizontal,
             children: [
               {
                 label: `No`,
                 onClick: () => {
                   form.close((error) => console.error(error));
-                  reject(true);
+                  reject(Error);
```

Comment:
I'm curious to understand what would happen if you call reject(Error); Should it be reject(new Error('Some message'));?

Reply:
You're correct! Doing this would reject the promise with the Error class, which doesn't make a lick of sense. I'm not sure why I changed that - will fix :)